GlobalExceptionFilter.App/HttpGlobalExceptionFilter.cs

```diff
@@ -49,5 +48,26 @@ public void OnException(ExceptionContext context)
             }
             context.ExceptionHandled = true;
         }
+
+        private void Log(ExceptionContext context)
+        {
+            if (context.Exception.GetType() == typeof(CustomApiException))
```

Comment:
if(context.Exception is CustomApiException customApiException)
{
}

Pattern matching is seems much more elegant